lang/da.js

```diff
@@ -173,7 +173,7 @@ export default {
 	"components.selection.selected-plus": "{count}+ valgt",
 	"components.selection-controls.label": "Handlinger for valg",
 	"components.sort.label": "Sortér",
-	"components.sort.text": "Sortér: {selectedItemText}",
+	"components.sort.text": "Sortér:{selectedItemText}",
```

Comment:
This change seems unexpected. Other places in the file keep the space after the colon so I think I'll change it back.